R/checks.R

```diff
@@ -286,6 +290,24 @@ check_workflow <- function(x, pset = NULL, check_dials = FALSE) {
     }
   }
 
+  tune_tbl <- tune_args(x)
+
+  if (nrow(tune_tbl) > nrow(pset)) {
+    not_tunable <- tune_tbl$id[!tune_tbl$id %in% pset$id]
+    not_tunable_tbl <- tune_tbl[tune_tbl$id %in% not_tunable, ]
+    msg <-
+      c("!" = "{cli::qty(not_tunable)}The parameter{?s} {.var {not_tunable}}
+               {?was/were} marked with `tune()`, though {?is/are} not supported for tuning.")
+    if (all(not_tunable_tbl$source == "model_spec")) {
```

Comment:
Maybe change this to `any()` rather than `all()`? i.e. if at least 1 parameter isn't tunable but came from the model specification, then it is probably worth talking about it

You should filter `not_tunable` down to `not_tunable[not_tunable_tbl$source == "model_spec"]` for this if you change it to `any()`

Reply:
Addressed in [84d08ed](https://github.com/tidymodels/tune/pull/549/commits/84d08ed54016169ebfd30597de11023801771849)!